pyproject.toml

```diff
@@ -13,7 +13,6 @@ dependencies = [
     "aiohttp>=3.6.2,<4.0.0",
     "maxminddb>=2.5.1,<3.0.0",
     "requests>=2.24.0,<3.0.0",
-    "setuptools>=60.0.0",
```

Comment:
I think we would probably want to keep the versioned requirement. I think that should just be moved up to `build-system.requires`.

Reply:
OK. It was false anyway, [support for pyproject.toml was only introduced in setuptools 61](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html).